Update 7: Premium (#256)
NilPointer-Software/bdfd-wiki: src/premium/awaitReactions.md

# $awaitReactions
*(for premium bots)*

> This function is used to await an reaction command.

Reviewer: ```suggestion
This function is used to await a reaction.
```

## Syntax
```
$awaitReactions[<command name;reaction>;...]
```

> ## Parameters
> - `command name`: It's the name which will be used inside `$reaction[]` callback.
> - `reaction`: It awaits the given emoji. Emoji must be either in Unicode or in discord emoji id format.

Reviewer: ```suggestion

## Parameters
- `command name`: The name which will be used inside the `$reaction[]` callback.
- `reaction`: The reaction to await. The reaction must be either a Unicode Emoji or a Discord custom emoji id.
```
